src/dashboard/src/components/api/views.py

```diff
@@ -80,18 +80,18 @@ def wrapper(request, *args, **kwargs):
 def authenticate_request(request):
     error = None
 
+    client_ip = request.META['REMOTE_ADDR']
+    whitelist = helpers.get_setting('api_whitelist', '').split()
+    if client_ip in whitelist:
+        LOGGER.debug('API called by trusted IP %s', client_ip)
+        return None
+
```

Comment:
The whitelist feature existed in Archivematica before there was the ability to generate API keys.  I think sevein is suggesting that Archivematica doesn't need to do both (enforce valid IP AND valid api key).

There are precedents - AWS doesn't have an ip whitelist for api access.  There are work arounds - it is easy enough to use a firewall to restrict access by ip.

I think this change is good (setting whitelist to '') - I think it is worth creating an issue to suggest removing the whitelist entirely - the new OR behaviour is probably more permissive than we want, so better to just have api key as requirement for auth.